cpu/cc2538/radio/cc2538_rf_netdev.c

```diff
@@ -59,6 +59,13 @@ static int _get(netdev_t *netdev, netopt_t opt, void *value, size_t max_len)
             }
             return sizeof(netopt_enable_t);
 
+        case NETOPT_CHANNEL:
+            if (max_len < sizeof(uint16_t)) {
+                return -EOVERFLOW;
+            }
+            *((uint16_t *)value) = (uint16_t)cc2538_get_chan();
+            return sizeof(uint8_t);
```

Comment:
This should return `sizeof(uint16_t)`, as the type you use above is `uint16_t`. According to the doc the latter is correct:

https://github.com/RIOT-OS/RIOT/blob/c5ebf60bf286a98f6ecd5e2acc97fba8e4dc5065/sys/include/net/netopt.h#L43-L45

Reply:
I don't know why I used `uint8_t` there. Thanks for spotting, fixed